src/tasks/minions/minigames/nexActivity.ts

```diff
@@ -62,7 +63,7 @@ export default class extends Task {
 			const loot = new Bank();
 			loot.add(NexMonster.table.kill(1, {}));
 			if (roll(80 + users.length * 2)) {
-				loot.add(randomItemFromArray(allNexItems), 1);
+				loot.add(randomItemFromArray(nexCL), 1);
```

Comment:
This is now dropping things that it shouldnt drop.

Reply:
Fixed. Checked other bosses, none of them have this logic for dropping uniques.